packages/graphback-datasync/src/services/DataSyncCRUDService.ts

```diff
@@ -12,16 +13,19 @@ export interface SyncList<T> {
  */
 export class DataSyncCRUDService<T = any> extends CRUDService<T> {
 
-  public constructor(modelName: string, db: DataSyncProvider, config: CRUDServiceConfig) {
-    super(modelName, db, config);
+  public constructor(model: ModelDefinition, db: DataSyncProvider, config: CRUDServiceConfig) {
+    super(model, db, config);
   }
   /**
    * sync
    * For delta queries
    */
-  public async sync(lastSync: Date, context: GraphbackContext, filter?: any, limit?: number): Promise<SyncList<T>> {
-
-    const res = await (this.db as DataSyncProvider).sync(lastSync, context, filter, limit);
+  public async sync(lastSync: Date, info?: GraphQLResolveInfo, filter?: any, limit?: number): Promise<SyncList<T>> {
+    let selectedFields: string[];
+    if (info) {
+      selectedFields = getSelectedFieldsFromResolverInfo(info, this.model)
+    }
```

Comment:
Should we pass "items" in this too? Cause a delta list will be sent.

See https://graphback.dev/docs/datasync/soft-delete

Reply:
/cc @ssd71

Reply:
Yes!!